20230727/s10-1690466917636/README.md

```diff
@@ -0,0 +1,64 @@
+# Scenario 8 - ALS Baseline with Sims-only, multiple k6 VUs
```

Comment:
This README is in s10 folder. Based on the pattern before, each folder name was representing corresponding scenario number. Here Scenario 8 is in s-10 folder.

Reply:
Fixed